src/dags/rdw.py

```diff
@@ -0,0 +1,167 @@
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.operators.python_operator import PythonOperator
+from common.db import DatabaseEngine
+from common.http import download_file
+from environs import Env
+from more_ds.network.url import URL
+from ogr2ogr_operator import Ogr2OgrOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from typing import Dict, Iterable
+
+
+from common import (
+    default_args,
+    SHARED_DIR,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    MessageOperator,
+    quote_string,
+)
+
+from sql.rdw import SQL_CREATE_TMP_TABLE, SQL_SWAP_TABLE
+
+# Source defaults to 1000 records per request.
+# There is no unlimited.
+DATA_LIMIT: int = 100_000_000
+DATA_SELECTIONS: Dict[str, Iterable[str]] = {
+    "basis": [
+        "kenteken",
+        "voertuigsoort",
+        "massa_rijklaar",
+        "toegestane_maximum_massa_voertuig",
+        "inrichting",
+        "datum_eerste_toelating",
+        "lengte",
+        "maximum_massa_samenstelling",
+    ],
+    "brandstof": ["kenteken", "brandstof_omschrijving", "emissiecode_omschrijving"],
+    "assen": ["kenteken", "aantal_assen", "as_nummer", "technisch_toegestane_maximum_aslast"],
+}
+
+dag_id: str = "rdw"
+description: str = (
+    "Contextuele informatie over een RDW geregistreerd voertuig op basis van het kenteken."
+)
+variables: Dict = Variable.get(dag_id, deserialize_json=True)
+endpoints: Dict = variables["data_endpoints"]
+tmp_dir: str = f"{SHARED_DIR}/{dag_id}"
+db_conn: object = DatabaseEngine()
+env: Env = Env()
+rdw_base_url: str = URL(env("AIRFLOW_CONN_RDW_BASE_URL"))
+
+
+with DAG(
+    dag_id,
+    description=description,
+    default_args=default_args,
+    user_defined_filters=dict(quote=quote_string),
+) as dag:
+
+    # 1. Post message on slack
+    slack_at_start = MessageOperator(
+        task_id="slack_at_start",
+        http_conn_id="slack",
+        webhook_token=slack_webhook_token,
+        message=f"Starting {dag_id} ({DATAPUNT_ENVIRONMENT})",
+        username="admin",
+    )
+
+    # 2. Create temp directory to store files
+    mkdir = BashOperator(task_id="mkdir", bash_command=f"mkdir -p {tmp_dir}")
+
+    # 3. Download csv
+    download_data = [
+        PythonOperator(
+            task_id=f"download_{resource}",
+            python_callable=download_file,
+            op_kwargs=dict(
+                url=URL(rdw_base_url)
```

Comment:
```suggestion
                url=rdw_base_url
```

goed punt van @gkoller : jouw `rdw_base_url` is al een `URL` dus hoef je deze niet om te zetten tot een `URL.

Reply:
:+1: Oeps, ik ben te gehaast vrees ik geweest